lib/Runtime/Library/ByteCodeString.h

```diff
@@ -0,0 +1,21 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+#pragma once
+
+namespace Js
+{
+    class ByteCodeString : public JavascriptString
```

Comment:
please add a banner comment for what this class does and why it exists

Reply:
(and, perhaps include some of the larger discussion above with seth and friends)